source/routing/asynchronous-routing.md

```diff
@@ -94,19 +94,19 @@ App.TardyRoute = Ember.Route.extend({
 });
 ```
 
-When transitioning into `TardyRoute`, the `model` hook will be called and
+When transitioning into `route:tardy`, the `model` hook will be called and
```

Comment:
why `router:tardy`? This is the factory name in the container, and users won't normally see this. They probably `transitionTo` or `link-to` the `tardy` route.